qutebrowser/browser/webengine/spell.py

```diff
@@ -0,0 +1,150 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2017 Michal Siedlaczek <michal.siedlaczek@gmail.com>
+
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Installing and configuring spell-checking for QtWebEngine."""
+
+import os
+from urllib.parse import urljoin
+from urllib.request import urlretrieve
+
+from PyQt5.QtCore import QLibraryInfo
+
+repository_url = 'https://redirector.gvt1.com/edgedl/chrome/dict/'
+
+
+class Language:
```

Comment:
qutebrowser recently [introduced](https://github.com/qutebrowser/qutebrowser/commit/3a5241b642da666e4517a32e0eb945254d86a6da) [attrs](http://www.attrs.org/en/stable/), I think this would be a great place to use it.